client-v2/src/lib/dnd/Level.tsx

```diff
@@ -146,6 +146,13 @@ class Level<T> extends React.Component<Props<T>> {
     }
 
     e.preventDefault();
+
+    // Reset the dragging state on drop
```

Comment:
I think this resetting is handled by the onDragEnd in the Root so needn’t happen here?

Reply:
You're absolutely right.